Sampler/BDE-Sampler.py

```diff
@@ -0,0 +1,19 @@
+# resize image to 64*64 to standardize the input for the training data
+# from skimage import data
+# from matplotlib import pyplot as plt
+from skimage import io
+from skimage.transform import resize
+from scipy import misc
+from PIL import Image
+import glob
+#CURE-TSR/Real_Train/*/*.bmp
+i=1
```

Comment:
Refer 
https://github.com/rachit-ranjan16/foodie/config 
for appConfig Extraction